Since version 2.5.3, Ruby compiled on SPARC is not passing one of its tests.
Problem is in load_from_binary function which causes Bus error:
.../ruby-25/ruby-2.5.3/test/ruby/test_iseq.rb:403: [BUG] Bus Error at 0x000000010d1de3ec

This is probably a Ruby core problem and was reported upstream:
https://bugs.ruby-lang.org/issues/15290

--- ruby-2.5.3/test/ruby/test_iseq.rb	2018-11-05 07:07:23.414144195 +0000
+++ ruby-2.5.3/test/ruby/test_iseq.rb	2018-11-05 07:06:11.814911953 +0000
@@ -391,17 +391,17 @@ class TestISeq < Test::Unit::TestCase
     }
   end
 
-  def test_to_binary_tracepoint
-    filename = "#{File.basename(__FILE__)}_#{__LINE__}"
-    iseq = RubyVM::InstructionSequence.compile("x = 1\n y = 2", filename)
-    iseq_bin = iseq.to_binary
-    ary = []
-    TracePoint.new(:line){|tp|
-      next unless tp.path == filename
-      ary << [tp.path, tp.lineno]
-    }.enable{
-      ISeq.load_from_binary(iseq_bin).eval
-    }
-    assert_equal [[filename, 1], [filename, 2]], ary, '[Bug #14702]'
-  end
+#  def test_to_binary_tracepoint
+#    filename = "#{File.basename(__FILE__)}_#{__LINE__}"
+#    iseq = RubyVM::InstructionSequence.compile("x = 1\n y = 2", filename)
+#    iseq_bin = iseq.to_binary
+#    ary = []
+#    TracePoint.new(:line){|tp|
+#      next unless tp.path == filename
+#      ary << [tp.path, tp.lineno]
+#    }.enable{
+#      ISeq.load_from_binary(iseq_bin).eval
+#    }
+#    assert_equal [[filename, 1], [filename, 2]], ary, '[Bug #14702]'
+#  end
 end
